java/middleware-commons/src/test/java/dev/sunbirdrc/registry/middleware/service/ConditionResolverServiceTest.java

```diff
@@ -44,7 +47,7 @@ public void shouldReturnTrueForValidExpression() throws IOException {
         try {
             resolve = conditionResolverService.resolve(getStudentJsonNode(), requester, condition, attributes);
         } catch (IOException e) {
-            e.printStackTrace();
+            logger.info("Exception occurred: {}", e.getMessage());
```

Comment:
logger.error and throw the exception

Reply:
Never log as log.info . This is an error. We will need the stack trace.  Use log.error and use exception utils for the message. Since the method signature has throws for IOException throw the exception from catch.